frontend/src/features/ControlUnit/components/ControlUnitDialog/ControlUnitContactList/index.tsx

```diff
@@ -18,16 +20,17 @@ import { Section } from '../shared/Section'
 import { TextareaForm } from '../shared/TextareaForm'
 
 import type { ControlUnitContactFormValues } from './types'
-import type { ControlUnit } from '../../../../../domain/entities/controlUnit'
 
 type ControlUnitContactListProps = {
   controlUnit: ControlUnit.ControlUnit
   onSubmit: (nextControlUnit: ControlUnit.ControlUnit) => any
 }
 export function ControlUnitContactList({ controlUnit, onSubmit }: ControlUnitContactListProps) {
+  const noEmailSubscriptionContactWarningBannerRankRef = useRef<number | undefined>(undefined)
```

Comment:
```suggestion
  const isNoEmailSubscriptionContactWarningBannerDisplayedRef = useRef<boolean>(false)
```
Je propose de simplifier et de cacher le concept de `rank`, inutilisé ici ?